sycl/doc/GetStartedGuide.md

```diff
@@ -222,13 +222,13 @@ tar -zxvf tbb*lin.tgz
 folder:
 ```bash
 ln -s /opt/intel/tbb_<tbb_version>/tbb/lib/intel64/gcc4.8/libtbb.so
-  /opt/intel/oclcpuexp/x64/libtbb.so
+  /opt/intel/oclcpuexp_<cpu_version>/x64
 ln -s /opt/intel/tbb_<tbb_version>/tbb/lib/intel64/gcc4.8/libtbbmalloc.so
-  /opt/intel/oclcpuexp/x64/libtbbmalloc.so
+  /opt/intel/oclcpuexp_<cpu_version>/x64
 ln -s /opt/intel/tbb_<tbb_version>/tbb/lib/intel64/gcc4.8/libtbb.so.2
-  /opt/intel/oclcpuexp/x64/libtbb.so.2
+  /opt/intel/oclcpuexp_<cpu_version>/x64
 ln -s /opt/intel/tbb_<tbb_version>/tbb/lib/intel64/gcc4.8/libtbbmalloc.so.2
-  /opt/intel/oclcpuexp/x64/libtbbmalloc.so.2
+  /opt/intel/oclcpuexp_<cpu_version>/x64
```

Comment:
Can we replace these four commands with one?
```bash
ln -s /opt/intel/tbb_<tbb_version>/tbb/lib/intel64/gcc4.8/ /opt/intel/oclcpuexp_<cpu_version>/x64
```

Reply:
@bader one way to merge into one common is to use glob expression while the other way is to list explicitly all the four files into one line. I actually prefer the current instruction since it is the most explicit and not too long, therefore suitable for a getting started guide.